BlueShift-iOS-SDK/BlueShiftRequestQueue.m

```diff
@@ -28,62 +28,44 @@ + (void)addRequestOperation:(BlueShiftRequestOperation *)requestOperation {
+                    isBatchEvent = NO;
                 }
-                if(entity != nil) {
-                    NSString *url = requestOperation.url;
-                    BlueShiftHTTPMethod httpMethod = requestOperation.httpMethod;
-                    NSDictionary *parameters = requestOperation.parameters;
-                    NSInteger nextRetryTimeStamp = requestOperation.nextRetryTimeStamp;
-                    NSInteger retryAttemptsCount = requestOperation.retryAttemptsCount;
-                    BOOL isBatchEvent = requestOperation.isBatchEvent;
-                    
-                    if ([BlueShiftNetworkReachabilityManager networkConnected] == NO)  {
-                        isBatchEvent = YES;
-                    }
-                    // Treat all the tracking events as non-batched events to stop them from getting batched
-                    NSString *trackURL = [BlueshiftRoutes getTrackURL];
-                    if ([requestOperation.url rangeOfString:trackURL].location != NSNotFound) {
-                        isBatchEvent = NO;
-                    }
-                    NSManagedObjectContext *context;
-                    if (isBatchEvent) {
-                        context = appDelegate.batchEventManagedObjectContext;
-                    } else {
-                        context = appDelegate.realEventManagedObjectContext;
-                    }
-                    if (context) {
-                        [context performBlock:^{
-                            HttpRequestOperationEntity *httpRequestOperationEntity;
-                            @try {
-                                httpRequestOperationEntity = [[HttpRequestOperationEntity alloc] initWithEntity:entity insertIntoManagedObjectContext:context];
-                                
-                                if(httpRequestOperationEntity != nil) {
-                                    [httpRequestOperationEntity insertEntryWithMethod:httpMethod andParameters:parameters andURL:url andNextRetryTimeStamp:nextRetryTimeStamp andRetryAttemptsCount:retryAttemptsCount andIsBatchEvent:isBatchEvent];
-                                    
-                                    if(!isBatchEvent) {
-                                        [BlueShiftRequestQueue processRequestsInQueue];
+                NSManagedObjectContext *context = appDelegate.realEventManagedObjectContext;
+                if (context) {
+                    @try {
+                        NSEntityDescription *entity;
+                        entity = [NSEntityDescription entityForName:kHttpRequestOperationEntity inManagedObjectContext:context];
+                        if(entity != nil) {
+                            [context performBlock:^{
+                                @try {
+                                    HttpRequestOperationEntity * httpRequestOperationEntity = [[HttpRequestOperationEntity alloc] initWithEntity:entity insertIntoManagedObjectContext:context];
```

Comment:
`insertIntoManagedObjectContext` or `inManagedObjectContext`?

Reply:
`insertIntoManagedObjectContext`  is correct. `initWithEntity:insertIntoManagedObjectContext:` is a core data method.

Reply:
`insertIntoManagedObjectContext`  is correct. `initWithEntity:insertIntoManagedObjectContext:` is a core data method.